external/VideoStreamer/build/VideoStreamer/VideoStreamer.vcxproj

```diff
@@ -0,0 +1,189 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup Label="ProjectConfigurations">
+    <ProjectConfiguration Include="Debug|Win32">
+      <Configuration>Debug</Configuration>
+      <Platform>Win32</Platform>
+    </ProjectConfiguration>
+    <ProjectConfiguration Include="Release|Win32">
+      <Configuration>Release</Configuration>

```

Comment:
No ARM or ARM64 support?

Reply:
Ffmpeg.nightly nuget package does not support ARM. Eventually we will either need to write our own packetizer or build our own nuget feed for ffmpeg arm32/64+x86/64 libraries